### DISCUSSION TITLE
Déterrage de Topics;  Passage de variable entre scènes

### DISCUSSION THREAD

**Discussion by JYK-WIT** [witsa/synapps#519] [Q&A] [answered]
Bonjour,
J'ai une scène générique qui doit être accessible pas plusieurs autres. Chaque scène doit envoyer au moins une variable à celle-ci.
![image](https://user-images.githubusercontent.com/100723121/178099050-6e160082-60b3-4fc7-bce2-19e9308bcceb.png)
Comment faire ?
Merci

**Comment by RaphaelBolier** [comment]
Hello, afin de passer des variables à la scène il est possible de passer des "paramètres de scène" autrement dit, de créer des additionnelles directement sur la scène elle-même !

![image](https://user-images.githubusercontent.com/55784043/178214247-581107de-7c20-4b6d-9ec3-13f62e26e678.png)

Ci-dessus, l'endroit ou les paramètres de scène doivent être ajoutés à la scène pour les utiliser ensuite la ou il est nécessaire.

**Comment by WITNicolas** [comment]
Autre chose qui a son importance, il faut que dans l'Acteur Ecran qui va afficher la Scene Chauffage Digesteur, tu es aussi la même aditionnelle (si je ne me trompe pas bien sûr).

**Comment by JYK-WIT** [comment, marked as answer]
Merci à tous, ça marche, avec les additionnelles dans chaque scène.
